concourse/steps/release.py

```diff
@@ -740,6 +740,69 @@ def apply(self):
         logger.info('successfully posted contents to slack')
 
 
+class MergeReleaseToDefaultStep(TransactionalStep):
+    def __init__(
+        self,
+        git_helper: GitHelper,
+        release_tag,
+        repository_branch,
+    ):
+        self.git_helper = not_none(git_helper)
+        self.release_tag = not_none(release_tag)
+        self.repository_branch = not_none(repository_branch)
+
+    def name(self):
+        return 'Merge release-commit to default branch'
+
+    def validate(self):
+        not_none(self.context().release_commit)
+
+    def apply(self):
+        # clean repository if required
+        worktree_dirty = bool(self.git_helper._changed_file_paths())
+        if worktree_dirty:
+            self.git_helper.repo.head.reset(working_tree=True)
+
+        default = self.git_helper.repo.head
+
+        def create_merge_commit():
+            merge_base = self.git_helper.repo.merge_base(
+                default,
+                self.context().release_commit,
+            )
+
+            self.git_helper.repo.index.merge_tree(
+                rhs=self.context().release_commit,
+                base=merge_base,
+            )
+
+            return self.git_helper.index_to_commit(
+                message=f'Merge release-commit from tag {self.release_tag}',
+                parent_commits=(
+                    default.commit,
+                    self.context().release_commit.commit,
+                ),
+            )
+
+        try:
+            merge_commit = create_merge_commit()
+
+            self.git_helper.push(
+                from_ref=merge_commit,
+                to_ref=self.repository_branch
+            )
+
+        except:
+            logger.warning(
+                f'Merging release-commit from tag {self.release_tag} failed.'
+                f'Manual actions might be required.'
```

Comment:
nit: this warning "manual actions might be required" is not helpful. either be more specific (especially w.r.t. what this means), or omit it and only leave the first line